mobile/test/common/integration/cds_integration_test.cc

```diff
@@ -0,0 +1,55 @@
+#include "envoy/config/bootstrap/v3/bootstrap.pb.h"
+#include "envoy/service/runtime/v3/rtds.pb.h"
+
+#include "test/common/integration/xds_integration_test.h"
+#include "test/test_common/utility.h"
+
+#include "gtest/gtest.h"
+
+namespace Envoy {
+namespace {
+const char ClusterName[] = "newcluster";
+
+class CdsIntegrationTest : public XdsIntegrationTest {
+public:
+  CdsIntegrationTest() {
+    use_lds_ = false;
+    default_request_headers_.setScheme("http");
+  }
+
+  void createEnvoy() override {
+    sotw_or_delta_ = sotwOrDelta();
+    const std::string api_type = sotw_or_delta_ == Grpc::SotwOrDelta::Sotw ||
+                                         sotw_or_delta_ == Grpc::SotwOrDelta::UnifiedSotw
+                                     ? "GRPC"
+                                     : "DELTA_GRPC";
+    builder_.addCdsLayer(1);
```

Comment:
add `/*timeout_seconds=*/` to the `1` so that the reader of the test can clearly know what that param value means